common-tasks/managing-appvm-shortcuts.md

```diff
@@ -16,7 +16,7 @@ Clicking on such shortcut runs the assigned application in its AppVM.
 
 ![dom0-menu.png"](/attachment/wiki/ManagingAppVmShortcuts/dom0-menu.png)
 
-To make newly installed applications show up in the menu, use the **qvm-sync-appmenus** command (Linux VMs do this automatically):
+To make newly installed applications show up in the menu, use the `qvm-sync-appmenus` command (Linux VMs do this automatically):
 
```

Comment:
I would clarify that this is for newly installed applications via the OS's package manager.

Reply:
Done, thanks!

Reply:
I liked what you did with the example file so included it too.